File: tests/benchmarks/cases/container_pipeline/dotnet/benchmark_case.cs
namespace BenchmarkRunner;

internal static class ContainerPipelineCase
{
    public const string Name = "container_pipeline";
    public const string PassBanner = "BENCH_CONTAINER_PIPELINE_PASS";

    public static long Run(int scale) => BenchmarkSupport.ContainerPipeline(scale);
}

File: tests/benchmarks/cases/container_pipeline/java/benchmark_case.java
final class ContainerPipelineCase {
    static final String NAME = "container_pipeline";
    static final String PASS_BANNER = "BENCH_CONTAINER_PIPELINE_PASS";

    private ContainerPipelineCase() {}

    static long run(int scale) {
        return BenchmarkSupport.containerPipeline(scale);
    }
}
